api/axis.py

```diff
@@ -963,7 +982,7 @@ def extract_bibliographic(self, element, ns):
             data_source=DataSource.AXIS_360,
             title=title,
             language=language,
-            medium=Edition.BOOK_MEDIUM,
+            medium=medium,
```

Comment:
I'm missing a test for this bit because I don't have a real availability document for an audiobook to use in the test.